### PR TITLE
Disallow "weak" crypto algorithms for `SCHANNEL_CRED`.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

**Comment by frewsxcv** [comment]
Was looking at [another implementation](https://github.com/wbond/oscrypto/blob/01db59353d98c8b97318b33830302f60adb3e013/oscrypto/_win/tls.py#L232) and noticed this was an option

File: src/lib.rs [hunk 1]
```diff
@@ -167,6 +167,7 @@ impl SchannelCredBuilder {
             let mut handle = mem::uninitialized();
             let mut cred_data: SCHANNEL_CRED = mem::zeroed();
             cred_data.dwVersion = SCHANNEL_CRED_VERSION;
+            cred_data.dwFlags = schannel::SCH_USE_STRONG_CRYPTO;
```
Comment (sfackler) [inline src/lib.rs]: Is this not reexported at the winapi crate root? AFAIK everything should be.

Comment (frewsxcv) [inline src/lib.rs]: It probably is, though I personally like the namespacing, but I'll change it

Comment (frewsxcv) [inline src/lib.rs]: https://github.com/sfackler/schannel-rs/pull/5/commits/dbfacd397d9386bfa55e5a974aafc1a05fd240c7

**Comment by sfackler** [comment]
LGTM but needs a rebase

**Comment by frewsxcv** [comment]
Rebased

**Pull request merged**